hq/app/logic/DocumentUtil.scala

```diff
@@ -0,0 +1,26 @@
+package logic
+
+import com.vladsch.flexmark.html.HtmlRenderer
+import com.vladsch.flexmark.parser.Parser
+import com.vladsch.flexmark.util.options.MutableDataSet
+import java.io.InputStream
+import play.twirl.api.Html
+
+object DocumentUtil {
+
+  def convert(markdownfile: String): Option[Html] = {
+    val filename = s"/$markdownfile.md"
+    var is = getClass.getResourceAsStream(filename)
```

Comment:
Make sure this is safe. Your old version had a whitelist of possible documentation paths and I think that is probably worth keeping.

For example, if the conf directory is on the classpath an attacker could use this to read the production config file.

Reply:
Only files ending .md can be fetched; is that sufficient whitelisting?